src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -191,6 +220,12 @@ function sealToken(name, playerColor)
           if name == "Bless" or name == "Curse" then
             blessCurseManagerApi.sealedToken(name, guid)
           end
+          --destroy XML on just covered token
+          if #sealedTokens > 1 then
+            local coveredToken = getObjectFromGUID(sealedTokens[#sealedTokens - 1])
```

Comment:
What happens if the token doesn't exist? For example because it was manually thrown back into the bag?